benchmarker/models/gemini/agent.py

```
@@ -0,0 +1,286 @@
+                    )
+                )
+            elif function_call.name in self._function_handlers:
+                handler = self._function_handlers[function_call.name]
+                result = handler(args)
+                function_responses.append(
+                    self._function_response_from_state(
+                        function_call.name, result, extra_fields
+                    )
+                )
+            else:
+                raise ValueError(f"Unsupported function: {function_call.name}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)